zombienet_tests/misc/0003-parathreads.toml

```diff
@@ -0,0 +1,32 @@
+[settings]
```

Comment:
are there any tests for this or just this config?

Reply:
There are no tests for this, this was included while developing for quick setup and was handy to have on the branch.
Should I remove it?

Reply:
Thanks for the clarification. Just wasted to confirm whether it was a manual setup config or the test was not committed by accident :) No need to remove it.